infrastructure/running-a-validator/onboarding-and-offboarding/set-up-validator.md

```diff
@@ -277,10 +276,7 @@ If you'd like to speed up the process further, you can use a database snapshot.
 !!!warning
     Although snapshots are convenient, syncing from scratch is recommended for security purposes. If snapshots become corrupted and most nodes rely on them, the network could inadvertently run on a non-canonical chain.
 
-??? info "Why am I unable to synchronize the chain with 0 peers?"
-    Make sure you have libp2p port `30333` activated. It will take some time to discover other peers over the network.
-
-    ![Terminal logs showing 0 peers](/images/infrastructure/running-a-validator/onboarding-and-offboarding/set-up-validator/set-up-a-validator-01.webp)
+If you are unable to synchronize the chain due to having zero peers, make sure you have libp2p port `30333` activated. It will take some time to discover other peers over the network.
```

Comment:
This seems like maybe it should be a tip, but I don't like the back to back admonitions haha so this works.

It might be helpful to still showcase what the terminal logs look like with 0 peers, though ideally, this should be displayed using a terminal element instead. Regardless, if the image is not being used (and is not used elsewhere), please delete it and update the numbering of the remaining images. For example, if `set-up-a-validator-01.webp` is removed, renumber the remaining images to maintain a sequential order starting from 1.